### ISSUE TITLE
Typo in the docs

### ISSUE THREAD

**Issue by mattdiamond** [emmanueltouzery/prelude-ts#62]
I believe this text should say "But if it was a Right..."

https://github.com/emmanueltouzery/prelude-ts/blob/7338785a9cc5f368527f2d19d9bab0aa3afc13e7/src/Either.ts#L704

**Comment by emmanueltouzery** [comment]
that's correct, thank you! I'll apply this change the next time i make a change in prelude, alternatively PR welcome!
